.github/workflows/nodejs.yml

```diff
@@ -31,8 +31,12 @@ jobs:
           node-version: 12.x
       - name: Install dependencies
         run: npm ci
+      - name: Set deploy path env variable
+        run: echo '::set-env name=BASE_URL::auth-landing'
       - name: Generate production build
-        run: npm run export
+        env:
+          BASE_URL: ${{ env.BASE_URL }}
```

Comment:
We should declare `BASE_PATH` on the job itself so that it is shared across steps.

```yaml
deploy-master:
  ...
  env:
    BASE_PATH: "/auth-landing"
  ...
  steps:
    ...
    - name: Generate production build
       run: npm run export -- --basepath='$BASE_PATH'
    ...
       FOLDER: __sapper__/export/${{ env.BASE_PATH }}
```